views/media/field-row.tt

```diff
@@ -0,0 +1,32 @@
+[% DEFAULT
+    labelSize = 2
+    fieldSize = 12 - labelSize
+    type      = "text"
+%]
+<div class="row">
+    <div class="large-[% labelSize %] columns">
+        <label for="[% name %]">[% label %]</label>
+    </div>
+    <div class="large-[% fieldSize %] columns">
```

Comment:
Having the label and the form element in two different divs looks rather odd in my browser, and probably doesn't make sense from a semantic standpoint either (although that's not my area of expertise and I am open to counterarguments...)  But I do like that you made this a reusable element!